src/scheduler.rs

```diff
@@ -54,7 +56,10 @@ async fn inbox_loop(ctx: Context, started: Sender<()>, inbox_handlers: ImapConne
 
     let ctx1 = ctx.clone();
     let fut = async move {
-        started.send(()).await;
+        started
+            .send(())
+            .await
+            .expect("inbox loop, missing started receiver");
```

Comment:
Same for all `expect` calls, can they be replaced with warnings written into logs? This way if someone accidentally kills the other side of the channel, we get an app that does not do networking, but still does not crash and allows to collect the log with these errors.

Reply:
I am worried that if we don't panic, the app will be in a state in which it is unclear what is happening to the user, as it might seem to be working, except no incoming or outgoing messages are happening. So I would rather crash the app, ensuring it gets restarted and we get a crash report hopefully

Reply:
Crash report will be hard to debug in this case. If nothing is sent/received, user will be able to send us a log, which is much more useful. Maybe throw an `error!`?